console/v1/0000_10_consoleplugin.crd.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1186
+    description: Extension for configuring openshift web console plugins.
+    displayName: ConsolePlugin
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: consoleplugins.console.openshift.io
+spec:
+  group: console.openshift.io
+  names:
+    kind: ConsolePlugin
+    listKind: ConsolePluginList
+    plural: consoleplugins
+    singular: consoleplugin
+  scope: Cluster
+  versions:
+    - name: v1
```

Comment:
The CRD we install in the cluster needs to contain both schemas, v1 and v1alpha1. V1 need to be `storage: false`. Can you add both schemas here and to the CRD in the v1alpha1 package please to make sure they match